staff/antonio-izquierdo/challenges/calculattor-app/README.md

```diff
@@ -0,0 +1,16 @@
+# Calculator App
+
+Esta aplicación es una calculadora básica. Permite realizar las siguientes operaciones matemáticas: sumar, restar, multiplicar y dividir.
```

Comment:
I cannot understand Spanish. Would you mind it translating to English? Thanks mate!

Reply:
Ok! Thank U